Extensions/find_inactives.js

```diff
@@ -24,8 +24,8 @@ XKit.extensions.find_inactives = new Object({
 				"A month", "30",
 				"Two months", "60",
 				"4 months", "120",
-				"A year", "365",
-			],
```

Comment:
The commas were just unneeded and my IDE nagging me about them.  If it's part of the style we want to keep I can put them back in.